examples/chart/teleport-cluster/values.yaml

```diff
@@ -35,6 +35,12 @@ enterprise: false
 podSecurityPolicy:
   enabled: true
 
+# If enabled will create a Pod Disruption Budget
+# https://kubernetes.io/docs/concepts/workloads/pods/disruptions/
+podDisruptionBudget:
+  enabled: false
+  minAvailable: 1
```

Comment:
I think that this should be under the `highAvailability` section as it's only relevant for that type of deployment.